lightning/src/offers/static_invoice.rs

```diff
@@ -534,6 +543,13 @@ impl Writeable for StaticInvoice {
 	}
 }
 
+impl Readable for StaticInvoice {
```

Comment:
Since #3640 landed this should be `LengthReadable` instead

Reply:
Mh ok, this means that we should change also the `Writeable` to add the length? or how this will work?

Reply:
Ah there's no need for that, `LengthReadable` just means that static invoices can only be read from a length-limiting reader since they will always drain the reader when read.

Reply:
Thanks for explaining to me, I was just assuming the wrong semantics!